libevmasm/Instruction.h

```diff
@@ -89,6 +89,7 @@ enum class Instruction: uint8_t
 	CHAINID,			///< get the config's chainid param
 	SELFBALANCE,		///< get balance of the current account
 	BASEFEE,            ///< get the block's basefee
+	BLOBHASH,           ///< get the blob versioned hash
```

Comment:
I believe it's better to state that it is associated with the transaction, right? Because the `index` in `blobhash(index)` represents the index of the blob within the transaction. Although EIP-4844 doesn't provide details on this, upon reviewing it and other associated EIPs like [EIP-2718: Typed Transaction Envelope](https://eips.ethereum.org/EIPS/eip-2718), it appears to be the case. Initially, I thought it was related to a beacon block number, but it doesn't seem to be the case

Reply:
Right, my bad. It's per transaction.